backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -466,4 +471,22 @@ public FacilityStats getFacilityStats(@Argument UUID facilityId) {
   public UUID getPermissibleOrgId(UUID orgId) {
     return orgId != null ? orgId : getCurrentOrganization().getInternalId();
   }
+
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public List<UUID> getOrgAdminUserIds(UUID orgId) {
+    Organization org =
+        organizationRepository.findById(orgId).orElseThrow(NonexistentOrgException::new);
+    List<String> adminUserEmails = oktaRepository.fetchAdminUserEmail(org);
+
+    return adminUserEmails.stream()
+        .map(
+            email -> {
+              ApiUser foundUser =
+                  apiUserRepository
+                      .findByLoginEmail(email)
+                      .orElseThrow(NonexistentUserException::new);
```

Comment:
Do we want to throw an error here? Or return a list of users we DO have in our db? I'm asking since I'm not sure how many prod orgs have this problem but on dev7 it was common where a user was in Okta and not in our DB.

For example,
Test org with `internalId`:  `"737bd4f3-8e5d-4bb6-be01-32f72666ca8c"`

Reply:
Ahh good point! Will make this change